docs_src/OVMS/pipelinerun.md

```diff
@@ -20,25 +16,35 @@ You can run the pipeline script, `run.sh` with `--workload ovms` option, and the
     - dgpu.1
     - xeon
 3. [Optional parameters](#optional-parameters)
+4. [Environment Variables](../environment_variables.md)
  
-Run the command based on your requirement. You have to get your choices for #1, #2, #3 above to start the pipeline run, see [details](#run-pipeline-with-different-input-sourceinputsrc-types) section below.
+Run the command based on your requirement. You have to get your choices for #1-4 above to start the pipeline run, see [details](#run-pipeline-with-different-input-sourceinputsrc-types) section below.
 
-### Check successful pipeline run
-Once pipeline run has started, you will expect containers to be running, see [check for pipeline run success](#status-of-running-a-pipeline); For a successful run, see [sample output results](#sample-output).
+### Optional Parameters
 
-### Stop pipeline run
-You can call `make clean-ovms` to stop the pipeline and all running containers for ovms, hence the results directory log files will stop growing. Below is the table of make commands you can call to clean things up per your needs:
+The following are the optional parameters that you can provide as input to `run.sh`. Note that these parameters would affect the performance of the pipeline.
 
-| Clean Containers Options                                     | Command                            |
-| -------------------------------------------------------------| -----------------------------------|
-| clean instance-segmentation container if any                 | <pre>make clean-segmentation</pre>      |
-| clean grpc-go dev container if any                           | <pre>make clean-grpc-go</pre>      |
-| clean all related containers launched by profile-launcher if any | <pre>make clean-profile-launcher</pre>  |
-| clean ovms-server container                                 | <pre>make clean-ovms-server</pre> |
-| clean ovms-server and all containers launched by profile-launcher          | <pre>make clean-ovms</pre>         |
-| clean results/ folder                                        | <pre>make clean-results</pre>      |
+- `--classification_disabled`: Disables the classification process of image extraction. By default, the classification is enabled.
 
----
+- `--ocr_disabled`: Disables optical character recognition (OCR). By default, OCR is enabled.
+
+- `--ocr`: Provides the OCR frame internal value, such as `--ocr 5 GPU`. The default recognition interval value is 5. Note device equal to CPU is not supported when executing with a discrete GPU.
+
+- `--barcode_disabled`: Disables barcode detection. By default, barcode detection is enabled.
+
+- `--realsense_enabled`: Uses the Intel® RealSense™ Camera and provides the 12-digit serial number of the camera as an input to the `run.sh` script.
```

Comment:
interesting about this one, i think we kind of refactor this one already : we change the way to detect this case: @vli11

Reply:
so i think we can remove this

Reply:
@jim-wang-intel which one should we remove?

Reply:
i am suggesting to remove --realsense_enabled since we kind of figure out the realsense case now by this code logic here: https://github.com/intel-retail/automated-self-checkout/blob/main/run-ovms.sh#L40-L49